extensions/amp-analytics/0.1/session-manager.js

```diff
@@ -186,38 +195,36 @@ function getStorageKey(type) {
 }
 
 /**
- * @param {SessionInfoDef|string} storedSession
- * @return {SessionInfoDef|undefined}
+ * @param {*} storedSession
```

Comment:
When might this be of a different type?

Reply:
That's the type we get back from `storage-impl/get()`. I imagine it has to do with the fact that we can't guarantee that no one will tamper with LS.